### ISSUE TITLE
Possible to install on Windows

### ISSUE THREAD

**Issue by GitterHubber** [jjdejong/phpip-legacy#23]
Since this app works on Apache / Mysql / PHP. I guess there should be no issues with running this on windows correct? Are there any special considerations before getting it to run on windows?

**Comment by jjdejong** [comment]
If you are familiar with an AMP installation on Windows, sure. We can't guarantee there will be "no issues" though.

**Comment by GitterHubber** [comment]
I would just install an AMP stack like Xampp or Wamp. Having said that what kind of issues could one expect?
The app would be running in a browser. correct?

**Comment by jjdejong** [comment]
I didn't mean issues in running the app, I meant issues in installing and configuring for the first run. Every flavor of AMP stack has different default settings.

**Comment by papoteur-mga** [comment]
Hello,
Any feedback of such an installation will be welcome.

**Comment by GitterHubber** [comment]
> 
> 
> Hello,
> Any feedback of such an installation will be welcome.

Yes indeed, I just couldn't get down to do it. I will surely post my inputs as and when I do it.
Are there any screenshots of the UI. Would love to have a look at them before I do start something.

**Comment by papoteur-mga** [comment]
Hello GitterHubber,
There is some screenshots with explanations here: https://github.com/jjdejong/phpip/wiki/Overview

**Comment by GitterHubber** [comment]
Ok, So I installed it successfully on Windows, I just followed the instructions and everything went smoothly.
I already had a wamp server running on a Windows box so didn't have to do much more than follow the instructions for installation.
